karma.conf.js

```diff
@@ -0,0 +1,54 @@
+module.exports = function(config) {
+  // Browsers to run on Sauce Labs
+  var customLaunchers = {
```

Comment:
For all but chrome(because apparently its default version is `27` versus `latest`), thinking we should drop the [version string](https://github.com/karma-runner/karma-sauce-launcher#version) so that we're always testing on the latest browser.